schemas/pharmacyItem.py

```diff
@@ -0,0 +1,12 @@
+from pydantic import BaseModel
+
+
+class PharmacyItem(BaseModel):
```

Comment:
Some of the field of this schemas is nullable. You can look whether it's nullable or not in it's model or entity.